README.markdown

```diff
@@ -10,9 +14,22 @@ gem 'omniauth-fishbrain'
 
 ## Usage
 
+In production environments:
+
 ```
 use OmniAuth::Builder do
   provider :fishbrain, ENV.fetch('FISHBRAIN_CLIENT_ID'), ENV.fetch('FISHBRAIN_CLIENT_SECRET')
+  provider :fishbrain_id
+end
+```
+
+In development/test/staging environments:
+
+```
+use OmniAuth::Builder do
+  provider :fishbrain, ENV.fetch('FISHBRAIN_CLIENT_ID'), ENV.fetch('FISHBRAIN_CLIENT_SECRET'), 
+                       user_pool_id: 'eu-west-1_K2uP41DlP' 
```

Comment:
the site name also needs to be set for staging environments

```
client_options: {
  site: 'https://accounts-staging.fishbrain.com',
}
```

Reply:
Good spot. Added (along with mention of support for path_prefix too.